client/superdesk.config.js

```diff
@@ -55,7 +55,10 @@ module.exports = function(grunt) {
         },
 
         monitoring: {
-            'scheduled': true
+            'scheduled': true,
```

Comment:
maybe the config could be something like:
```
monitoring: {
  scheduled: {
    sort: ['publish_schedule'],
  }
}
```
so we could add other options later, like max items, default view type etc

Reply:
Interesting, I'll check where `monitoring.scheduled` is used so I make sure it only checks it's defined, and not a boolean.

Reply:
Even if it checks for boolean, you can change it. It's not fun due to the number of repositories, but you can do it :) By the way, why is the field name config inside an array? can you sort by multiple fields at once?

Reply:
@tomaskikutis of course, I meant I was gonna change it. Yes, you can have multiple field to sort. You can't sort all of them at once but you can choose on of them from a dropdown

Reply:
Ok I made it so we can keep the `scheduled: true` and also use an object if we want to sort:

```js
        monitoring: {
            stage: {
                sort: [ 'headline' ]
            },
            scheduled: {
                sort: [ 'publish_schedule', 'headline' ]
            },
        }
```